cli/src/commands/application/init.rs

```diff
@@ -49,6 +49,9 @@ pub async fn handle_application_init(context: Context) -> Result<bool, WKCliErro
         .prompt()?;
     }
 
+    eprintln!();
+    eprintln!("Next is to configure the prod namespace for your application.");
```

Comment:
why not use this way to reduce the number of lines

```suggestion
    eprintln!("\nNext is to configure the prod namespace for your application.\n");
```